### ISSUE TITLE
Pretrained Model for preprocessed_DIP_IMU_c?

### ISSUE THREAD

**Issue by colincqian** [jyf588/transformer-inertial-poser#13]
Hello Yifeng,

Thank you for this excellent work. I am wondering whether u could provide the model and pipeline for using pre-trained model to label pseudo ground-truth SBPs for the DIP motions? I captured some real-world IMU data, and I would like to test it on TIP. Thank you in advance for your help.

**Comment by jyf588** [comment]
Hello, sorry I don't have time to clean up that part of code right now. Conceptually you should be able to just take my pre-trained model, run on your own data (using [this](https://github.com/jyf588/transformer-inertial-poser/blob/master/offline_testing_simple.py#L78) function), to generate the SBPs? I don't foresee you need a complex "pipeline". 

I'm assuming the IMU data you collect yourself do not have GT root translation (otherwise you can just label SBPs using data-gen-and-viz-bullet-new.py). Note that I only use my pre-trained model to label pseudo ground-truth SBPs, but not pseudo root translation. The root translation is [still marked as nan](https://github.com/jyf588/transformer-inertial-poser/blob/master/preprocess_and_combine_syn_amass.py#L62) and ignored in the loss function. This is because the models tend to have largest errors on root translation, where as SBP predictions are usually more robust.
